README.md

```diff
@@ -286,6 +286,7 @@ MONGOMS_USE_LINUX_LSB_RELEASE # Only try "lsb_release -a"
 MONGOMS_USE_LINUX_OS_RELEASE # Only try to read "/etc/os-release"
 MONGOMS_USE_LINUX_ANYFILE_RELEASE # Only try to read the first file found "/etc/*-release"
 MONGOMS_ARCHIVE_NAME="mongodb-linux-x86_64-4.0.0.tgz" # Specify what file / archive to download
+SKIP_OS_RELEASE # ignore error thrown in "tryOSRelease"
```

Comment:
Suppose that it's a typo. I think it should be `MONGOMS_SKIP_OS_RELEASE`

Reply:
yes definitely, this was an result of the copy-pasting i had done for the enum after i noticed this one was missing in the readme

Reply:
fixed in e35249523a275fe7b83a31420aa3c0210349cb60